src/app/pages/BuyPage/components/Tabbar/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+import { colors } from '../../../../constants';
+import styles from './index.module.scss';
+interface ActiveProps {
+  setSelectedTab: (e: boolean) => void;
+  selectedTab: boolean;
+}
+
+const Bar = styled.div<{ active: boolean }>`
```

Comment:
Pls replace this styled-components with the one using module css

Reply:
fixed